packages/sdk/src/swap/assets.ts

```diff
@@ -8,63 +7,74 @@ import {
   isTestnet,
 } from '@/shared/enums';
 import type { AssetData } from './types';
+import type { Environment } from '../rpc';
 
-export const eth$: (network: ChainflipNetwork) => AssetData = (network) => ({
+type AssetFn = (
+  network: ChainflipNetwork,
+  env: Pick<Environment, 'ingressEgress' | 'swapping'>,
+) => AssetData;
+
+export const eth$: AssetFn = (network, env) => ({
   id: Assets.ETH,
   chain: Chains.Ethereum,
   contractAddress: undefined,
   decimals: assetDecimals[Assets.ETH],
   name: 'Ether',
   symbol: 'ETH',
   isMainnet: !isTestnet(network),
-  minimumDepositAmount: getMinimumDepositAmount(network, Assets.ETH),
-  minimumSwapAmount: getMinimumSwapAmount(network, Assets.ETH),
+  minimumDepositAmount:
+    env.ingressEgress.minimumDepositAmounts.Ethereum.ETH.toString(),
+  minimumSwapAmount: env.swapping.minimumSwapAmounts.Ethereum.ETH.toString(),
```

Comment:
I doubt we need both of these because the `minimumSwapAmount` should always be greater than the deposit amount

Reply:
i think the reason for the separate `minimumDepositAmount` is that its more expensive for the protocol to "fetch" the funds than just swap them. but could remember this wrong

Reply:
I think for LP deposit channels the minimum deposit amount is more interesting, but here it should be redundant.